projects/WeTek_Hub/linux/linux.aarch64.conf

```diff
@@ -34,7 +39,7 @@ CONFIG_CROSS_COMPILE=""
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="@DISTRONAME@"
-CONFIG_SWAP=y
+# CONFIG_SWAP is not set
```

Comment:
You can check here: https://github.com/codesnake/LibreELEC.tv/commits/le_hosted_aml_kernel. I was about to PR the same changes as you, but you was the fisrt :). I also synced with .config, but CONFIG_SWAP didn't change.